src/DomainHost.sol

```diff
@@ -336,38 +366,49 @@ abstract contract DomainHost {
         emit Cage();
     }
 
-    /// @notice Set this domain's cure value
+    /// @notice Process this domain debt
     /// @param _lid Local ordering id
-    /// @param value The value of the cure [RAD]
-    function _tell(uint256 _lid, uint256 value) internal ordered(_lid) {
+    /// @param _debt The value of debt [RAD]
+    function _tell(uint256 _lid, uint256 _debt) internal ordered(_lid) {
         require(live == 0, "DomainHost/live");
-        require(!cureReported, "DomainHost/cure-reported");
-        require(_divup(value, RAY) <= grain, "DomainHost/cure-bad-value");
+        require(!debtReported, "DomainHost/debt-reported");
+
+        uint256 wad = grain + ddai;
+        uint256 debtW = _divup(_debt, RAY);
+        if (wad > debtW) {
+            unchecked {
+                wad = wad - debtW;
+            }
+            dai.transferFrom(address(escrow), address(this), wad);
+            daiJoin.join(address(vow), wad);
+        }
 
-        cureReported = true;
-        cure = value;
+        debt = _debt;
+        debtReported = true;
```

Comment:
Old code was 
```
        cureReported = true;
        cure = value;
```

and new code is in the reverse order.

I propose that we do it as:

```
        debtReported = true;
        debt = _debt;
```